src/styles/utils/variables.scss

```diff
@@ -0,0 +1,14 @@
+$maincolor: #060b35;
+$seccolor: #616070;
```

Comment:
If you use '-' this case for split words in variables so keep doing this for one code style in while project.
And also checklist point number 11